src/components/Article.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
```

Comment:
remove unused link import

Reply:
Sorry Hev, missed that! Will remove and merge.